recipes/ptmcmcsampler/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ptmcmcsampler" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptmcmcsampler-{{ version }}.tar.gz
+  sha256: 1f96ea651a3c3964a7e37ca3c3924aff83a552d0b278bfcc991af265d8739ae3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.16.3
+    - python >=3.6
+    - scipy >=1.2.0
+    - mpi4py >=3.0.3
```

Comment:
I see mpi4py is an optional/extras dependency upstream. Does *every* conda user of this package need to install mpi?

Reply:
I'm pretty new to conda, can we do optional requirements like in with pip? If not, what is the recommended way to do this?

Reply:
There's not great support for optional dependencies. One way is to just make it just generally required, like you've done. In that case it's always available, but *everyone*, even non-MPI users will end up with MPI installed.

The other option is to leave it off, and rely on users of MPI already having `mpi4py` installed or knowing enough to install it manually. Obviously, that has discoverability issues. So I leave it to you as the maintainer and package contributor what you think is best for the users of `ptmcmcsampler` or even just your own needs.

Reply:
I've removed it. I'll add some instructions on the github page about installing this dependency if needed on conda.